### ISSUE TITLE
Mask the input prompt

### ISSUE THREAD

**Issue by unlabeledData** [2018cx/Multi-Level-OT#6]
Original question:
During the data preparation, the [Multi-Level-OT] masks all input prompt by setting them as -100. I think it means only keep output tokens' logits during loss computation. Is my understanding correct? Could you share why you choose mask input prompt?

Now I know it widely used setting.
